db.sql

```diff
@@ -31,6 +31,7 @@ CREATE TABLE `conn_bundle` (
   `fw_build_id` int(11) DEFAULT NULL,
   `launch_description` varchar(512) DEFAULT NULL,
   `fw_digest` varchar(512) DEFAULT NULL,
+  `symkey` varchar(512) DEFAULT NULL,
   `create_date` datetime DEFAULT NULL,
```

Comment:
Not directly related to this PR, but it made me think for how long we should respect a given `client_id` .

For example, when looking for connections in the DB, we can add a `AND create_date > (NOW() - INTERVAL 1 HOUR)` (I'm sure the actual SQL syntax is wrong).

We can open another issue to discuss this.

Reply:
Yeah I'm thinking we will just clear out the DB periodically. I don't think there is any nice way for the KBC to tell the KBS that it is finished since new secret requests could come at any time.